modules/openapi-generator/src/main/resources/csharp/libraries/generichost/modelGeneric.mustache

```diff
@@ -347,4 +347,4 @@
 {{>validatable}}
 {{/parentModel}}
 {{/validatable}}
-    }
\ No newline at end of file
+    }
```

Comment:
Please remove the spacing changes in this file.

Reply:
Done.